www/src/theme.js

```diff
@@ -24,33 +24,30 @@ const colors = {
   darker: '#20232a', // really dark blue
   brand: '#61dafb', // electric blue
   brandLight: '#bbeffd',
+  brandOnWhite: '#10819b',
```

Comment:
nit `brandDark` ?

Reply:
this was actually an artefact, so now removed :)